receiver/nsxreceiver/metadata.yaml

```diff
@@ -0,0 +1,110 @@
+name: nsxreceiver
+
+resource_attributes:
+  nsx.node.name:
+    description: The name of the NSX Node.
+    type: string
+  nsx.node.id:
+    description: The ID of the NSX Node.
+    type: string
+  nsx.node.type:
+    description: The type of NSX Node.
+    type: string
+    enum:
+      - manager
+      - host
+      - edge
+  nsx.interface.id:
+    description: The name of the network interface.
+    type: string
+
+attributes:
+  direction:
+    value: direction
+    description: The direction of network flow
+    enum:
+      - received
+      - transmitted
+  disk:
```

Comment:
Any concern with this being a high cardinality attribute?

Reply:
Perhaps, I knew it would most likely be high cardinality but didn't think it would be too much of an issue. Do you know of any implications that could arise out of the cardinality?

Reply:
Attributes are used to slice data, so high cardinality attributes can result in an unreasonable number of slices. This can be a problem for backends as well as front ends (graphs with too many series).

Reply:
Here's a good explanation of the impact it can have: https://cloud.google.com/monitoring/api/v3/metric-model#cardinality

Reply:
Summing at the node level does seem like it most likely solves any cardinality issues, but you lose some granularity.

Reply:
Those are the two solutions I am aware of. Perhaps the simpler solution is better for now. Potentially some other forms of aggregation could be used here (eg. lowest available space on any disk, etc)

Reply:
My most recent commit has made the metrics that used this attributed aggregated per node rather than per file-system on the node.